### DISCUSSION TITLE
How to specify data type for database in remult?

### DISCUSSION THREAD

**Discussion by CKylinMC** [noam-honig/radweb#219] [Q&A] [answered]
I have a field with more than 255 characters as its content, and I want to use postgres as the database. But when I store this field it will throw `value too long for type character varying(255)`, I know `TEXT` type of postgres will work in this case, but how to specify it?

**Comment by noam-honig** [comment, marked as answer]
Hi, 

The 255 limit compes from `knex` 

If you're using Postgres, I recommend that you'll use the `remult/postgres` data provider, it would use `varchar` by default that doesn't have that limitation
```ts
import express from "express"
import { remultExpress } from "remult/remult-express"
import { createPostgresConnection } from "remult/postgres"

const app = express()

const connectionString = "postgres://user:password@host:5432/database"

app.use(
  remultExpress({
    dataProvider:
      createPostgresConnection({
        connectionString // default: process.env["DATABASE_URL"]
      })
  })
)
```

It'll solve your problem.

If you still want to determine the dbType here's how you can do that :
```ts
@Fields.string({
 valueConverter:{
  fieldTypeInDb:"the type you want"
 }
})
title = ""
```

But if I were you, I use the `remult/postgres`

**Reply by CKylinMC** [reply]
OK, I'll try `remult/postgres`, thanks!
